report/main.qmd

```diff
@@ -1 +1,166 @@
+
+It is recommended to fit GLS models separately for each of the three sites, with zero-calorie beverages percentage or sugary beverages percentage as the response variable. In addition to the reason mentioned in the second paragraph of section 3, another advantage of using percentage terms over raw sales is that we just need to fit models for either zero-calorie beverages percentage or sugary beverages percentage instead of for both. An increase in zero-calorie beverages percentage implies a decrease in sugary beverages percentage, vice versa.
+
+Regarding the predictor variables,  we recommend using `Intervention` and `DofW`. For `Intervention`, it is important to first rename some of the categories. For the observations made in the second pre-intervention period and the second follow-up period, the values of `Intervention` should be respectively replaced by something like "preint2" and "follow2". This way we are not mixing up the first and second pre-intervention and follow-up period in our analysis. Also, "wash" and "wash2" can be combined into one category as they are not of the main interest in the study. For `DofW`, Monday through Friday can be merged into a category called weekday, and Saturday and Sunday can be merged into a category called weekend. When adding text label of `DofW` to each of the data points in Figure 2, it can be seen that most of the valleys of the time series happen at weekends. Therefore, it is important to control for day of week when making comparisons between intervention and pre-intervention periods. In summary, the non-residual part of each site-specific model can be specified as ZeroCal_percent ~ Intervention_updated + DofW_updated or Sugary_percent ~ Intervention_updated + DofW_updated.
+
+When it comes to the residuals part of each site-specific model, it is suggested to compare four different correlation structure: independent, the first-order autoregressive process (AR(1)), the first-order moving-average process (MA(1)), and the combined autoregressive-moving-average processes (ARMA(p=1, q=1)). A GLS with an independent correlation structure is equivalent to a linear regression which assumes independence among residuals. A GLS with a AR(1) allows the residual at time point t to depend on the residual at the time point t-1. A GLS with a MA(1) is another way to capture temporal correlation among residuals, and ARMA(p=1, q=1) is the combination of AR(1) and MA(1) [@fox]. For each site, the GLS model with the smallest AIC values can be chosen as the final model. As a result, there will be three final models, one for each site, and the estimated coefficients of different interventions can be used for intervention effect assessment and comparison.
```

Comment:
It would help to provide more explanation for these four correlation structures and what they represent in terms of the dataset. For example, an AR(1) correlation structure means that the sales proportion on each day is correlated with the proportion of sales on the previous day.

You should also give more detailed explanations of what MA(1) means, as the description which you provided is very vague and the clients may not be familiar with these time series methods. 

Another possible consideration is to explain how the client can fit the specific hyperparameters of their correlation structures, such as deciding whether AR(2) may be better than AR(1).

Reply:
Additionally, you should specify how the AIC-based model selection method will work when you are fitting three separate models. It would make the most sense (to me) to use the same correlation structure for all three GLS models, but how do you combine the AIC of the three models if the three models do not have the same "optimal" correlation structure?